requirements.txt

```
@@ -1,3 +1,3 @@
 cython
 virtualenv
-git+https://github.com/learningequality/python-for-android@350d7158a0a35f578a95d50de969101579bbdc4f#egg=python-for-android
+git+https://github.com/learningequality/python-for-android@b0ad42809044635c31c7b421093ac546d5e0812d#egg=python-for-android
```

Since you merged the change I wanted:

```suggestion
git+https://github.com/learningequality/python-for-android@999271bab4450daf21fc4fd7ef2fe915be9cca1b#egg=python-for-android
```